docs/data/charts/funnel/funnel.md

```diff
@@ -10,7 +10,7 @@ components: FunnelChart, FunnelPlot
 
 ## Basics
 
-Funnel charts series should contain a data property containing an array of objects.
+Funnel charts series should contain a `data` property containing an array of objects.
```

Comment:
should, or must? (In other words, will it work without the data property?) Also, I think we should opt for a synonym for "property" so as not to confuse it with a "prop." Maybe "attribute"?